### PR TITLE
Add JS Proxy/Reflect compat data

### PR DESCRIPTION
**Pull request by Elchi3**

### CODE REVIEW

File: javascript/builtins/Proxy.json [hunk 1]
```diff
@@ -0,0 +1,874 @@
...
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "handler": {
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply javascript/builtins/Proxy.json]: I disagree. The "handler" itself is not really a feature, it is just an object where all the handlers are installed on. The table on this page will be an aggregation table of all the proxy handlers available. "Basic support" for an undefined amount of handlers would be very vague.

File: javascript/builtins/Proxy.json [hunk 2]
```diff
@@ -0,0 +1,874 @@
...
+          },
+          "set": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Proxy/handler/set",
+              "support": {
+                "webview_android": {
+                  "version_added": "56"
+                },
+                "chrome": {
+                  "version_added": "49"
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply javascript/builtins/Proxy.json]: It was true and recently someone changed it to 61. Let's be consistent and put 49.

File: javascript/builtins/Proxy.json [hunk 3]
```diff
@@ -0,0 +1,874 @@
...
+                },
+                "chrome_android": {
+                  "version_added": "49"
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply javascript/builtins/Proxy.json]: See comment above

**Comment by Elchi3** [comment]
Updated! Did you review Reflect? There were no comments on it.  (that's fine with me :)

File: javascript/builtins/Proxy.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (teoli2003) [reply javascript/builtins/Proxy.json]: Makes sense.

**Comment by renefb** [comment]
Hey guys! I would like to contribute to this issue, but I'm confused... Today (2018/05/03) the Proxy MDN's page still says "This compatibility table still uses the old format, because we haven't yet converted the data it contains", but it looks like you have been working on this above... So, the compatibility table still needs to be converted or not?

**Comment by Elchi3** [comment]
Do you have a link to the MDN page? Is it a translation that hasn't been updated yet?

**Comment by renefb** [comment]
Yeah, it's really a translation that hasn't been updated yet (https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Proxy)...
Sorry guys!

**Pull request merged**